bootstrap.sh

```diff
@@ -239,6 +239,13 @@ EOF
 
   echo "You may want to install the following:"
   sudo softwareupdate -l -a
+
+  git clone https://github.com/travis-ci/system-info.git
+  cd system-info
+  bundle install
+  bundle exec ruby exe/system-info
```

Comment:
I think dropping `--cookbooks-sha fafafa` and using the rest of what Dan suggested is _enough_ for now